token/token.go

```diff
@@ -122,6 +122,23 @@ func NewManager(config tokenManagerConfiguration) (Manager, error) {
 	return tm, nil
 }
 
+// NewFakeManager returns a new fake token Manager for handling fake tokens
+func NewFakeManager(config tokenManagerConfiguration) (Manager, error) {
```

Comment:
If you need to add/use any dev keys then use token.TokenManager from test/token instead.

Reply:
done!